samples/word/99-preview-apis/manage-styles.yaml

```diff
@@ -0,0 +1,357 @@
+            if (newStyleName == "") {
+              console.warn("Enter a style name to add.");
+              return;
+            }
+
+            const style = context.document.getStyles().getByNameOrNullObject(newStyleName);
+            style.load();
+            await context.sync();
+
+            if (!style.isNullObject) {
+              console.warn(
+                `There's an existing style with the same name '${newStyleName}'! Please provide another style name.`
+              );
+              return;
+            }
+
+            const newStyleType = ($("#new-style-type").val() as unknown) as Word.StyleType;
+            context.document.addStyle(newStyleName, newStyleType);
+            await context.sync();
+
+            console.log(newStyleName + " has been added to the style list.");
+          });
+        }
+
+        async function getProperties() {
+          // Gets the properties of the specified style.
+          await Word.run(async (context) => {
+            const styleName = $("#style-name-to-use").val() as string;
+            if (styleName == "") {
+              console.warn("Enter a style name to get properties.");
+              return;
+            }
+
+            const style = context.document.getStyles().getByNameOrNullObject(styleName);
+            style.load();
+            await context.sync();
+
+            if (style.isNullObject) {
+              console.warn(`There's no existing style with the name '${styleName}'.`);
+            } else {
+              style.font.load();
+              style.paragraphFormat.load();
+              await context.sync();
+
+              console.log(`Properties of the '${styleName}' style:`);
+              console.log(style);
+            }
+          });
+        }
+
+        async function applyStyle() {
+          // Applies the specified style to a paragraph.
+          await Word.run(async (context) => {
+            const styleName = $("#style-name-to-use").val() as string;
+            if (styleName == "") {
+              console.warn("Enter a style name to apply.");
+              return;
+            }
+
+            const style = context.document.getStyles().getByNameOrNullObject(styleName);
+            style.load();
+            await context.sync();
+
+            if (style.isNullObject) {
+              console.warn(`There's no existing style with the name '${styleName}'.`);
+            } else if (style.type != Word.StyleType.paragraph) {
+              console.log(`The '${styleName}' style isn't a paragraph style.`);
+            } else {
+              const body = context.document.body;
+              body.clear();
+              body.insertParagraph(
+                "Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.",
```

Comment:
I've always found this text about video kind of weird considering you can't add video through the JS APIs.

Reply:
Do you want us to change the setup text to use "Lorem ipsum" or some other content?

Reply:
Not sure if it's worth the effort, but something to consider in a future sample.